apps/health_campaign_field_worker_app/lib/pages/login.dart

```diff
@@ -191,11 +187,11 @@ class _LoginPageState extends LocalizedState<LoginPage> {
 
   FormGroup buildForm() => fb.group(<String, Object>{
         _userId: FormControl<String>(
-          // value: 'UAT-DISTRIBUTOR',
+          value: 'ANG-SUPERVISOR',
```

Comment:
@naveen-egov revert this file above changes are removing the version part from login screen

Reply:
Done